app/assets/javascripts/accordion.js

```diff
@@ -6,7 +8,205 @@ document.addEventListener('DOMContentLoaded', function (event) {
+                    parTitle.insertAdjacentHTML("afterend", '&nbsp;<span id="pubform-partitle" class="badge badge-info required-tag">required</span>');
+                    break;
+                }
+
+                case 'Book': {
+                    resetForm();
+                    console.log("Chose: Book");
+                    break;
+                }
+
+                case 'Book Chapter': {
+                    resetForm();
+                    console.log('Selected: Book Chapter');
+                    break;
+                }
+
+                case 'Conference Paper': {
+                    resetForm();
+                    console.log('Selected: Conference Paper');
+                    break;
+                }
+
+                case 'Poster': {
+                    resetForm();
+                    console.log('Selected: Poster');
+                    break;
+
+                }
+
+                case 'Presentation': {
+                    resetForm();
+                    console.log('Selected: Presentation');
+                    break;
+                }
+
+                case 'Report': {
+                    resetForm();
+                    console.log('Selected: Report');
+                    break;
+                }
+
+                case ' ': {
+                    resetForm();
+                    console.log('none selected');
+                    break;
+                }
+            }
+        }
+
+
+        function resetForm() {
+            //reset the form at the beginning every time we switch
+
+            //article reset
+            console.log('RESET FORM!');
+            confName[0].style.display = "block";
+            isbn[0].style.display = "block";
+            series[0].style.display = "block";
+            edition[0].style.display = "block";
+            var pubformPubver = document.getElementById("pubform-pubver");
+            if (pubformPubver) { pubformPubver.remove(); }
+            var pubformPubTitle = document.getElementById("pubform-partitle");
+            if (pubformPubTitle) { pubformPubTitle.remove(); }
+        }
+    }
 });
+
+
+function validateForm() {
+    var validationLabel = document.getElementById("publication_content_genre").selectedOptions[0].label;
+    // console.log("validating " + validationLabel);
+
+    switch (validationLabel) {
+        case 'Article': {
+
+            var pubverCheck = document.getElementById("publication_publisher_version").selectedOptions[0].label;
+            if (pubverCheck == ' ') {
+                var pubverLabel = "Publisher Version";
+                validateModal(pubverLabel);
+                return false;
+            }
+            var parTitle = document.getElementById("publication_parent_title").value;
+            if (!parTitle) {
+                var parTitleLabel = "Parent Title";
+                validateModal(parTitleLabel);
+                return false;
+            }
+
+            if (pubverCheck != ' ' && parTitle) {
```

Comment:
Same as above, add a default case for all other cases with the same instructions